autoreject/tests/test_ransac.py

```diff
@@ -50,12 +53,146 @@ def test_ransac():
 
     # should not contain both channel types
     picks = mne.pick_types(epochs.info, meg=True, eeg=False, stim=False,
-                           eog=False, exclude=[])
+                            eog=False, exclude=[])
     ransac = Ransac(picks=picks)
     pytest.raises(ValueError, ransac.fit, epochs)
 
     # should not contain other channel types.
     picks = mne.pick_types(raw.info, meg=False, eeg=True, stim=True,
-                           eog=False, exclude=[])
+                            eog=False, exclude=[])
     ransac = Ransac(picks=picks)
     pytest.raises(ValueError, ransac.fit, epochs)
+
+
+        
+def test_ransac_multiprocessing():
+    """test on real data with 5 random channels augmented with strong
+    50 Hz line noise."""
+    data_path = sample.data_path()
+    raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+    raw = io.read_raw_fif(raw_fname, preload=False)
+
+    raw.del_proj()
+
+    event_id = {'Visual/Left': 3}
+    tmin, tmax = -0.2, 0.5
+
+    events = mne.find_events(raw)
+    epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
+                        baseline=(None, 0), reject=None, preload=True)
+    # see if results are consistent when using different n_jobs
+    # amplify some of the channels with noise as sanity check
+    np.random.seed(0)
+    
+    # create some artifically noisy channels with strong line noise
+    mag_idx = mne.pick_types(epochs.info, meg='mag')
+    noisy_idx = np.random.choice(mag_idx, 5, replace=False)
+    noisy_idx = sorted(noisy_idx)
+    
+    times = np.arange(0, epochs.tmax-epochs.tmin + 1/epochs.info['sfreq'], 
+                      1/epochs.info['sfreq'])
```

Comment:
do you have a flake8 linter? It seems to not conform to pep8

Reply:
I should do that indeed. No I don't have it active right now